content/community/steering/index.md

```diff
@@ -52,14 +54,12 @@ Please contact the committee member leading the project if you would like to lea
 * Global Science - Andrew Lonie
 * Small scale servers - Hans-Rudolf Hotz
```

Comment:
Should we add the new subcommittee on mentoring led by @bebatut?

Reply:
Not until Berenice is ready? I think she made it clear at the meeting that it wasn't quite time - too busy...?

Reply:
It's a really important development but let's announce it when it happens!